src/main/java/teammates/common/datatransfer/questions/FeedbackMsqQuestionDetails.java

```diff
@@ -479,9 +479,9 @@ public String getQuestionAdditionalInfoHtml(int questionNumber, String additiona
             optionListHtml.append(optionHelpText);
         }
 
-        if (numOfMsqChoices > 0) {
+        if (!msqChoices.isEmpty()) {
```

Comment:
Actually I feel that for clarity, this whole block should be an else-if of the if statement just above. It should only run for non-generated options.